### ISSUE TITLE
autograd blitz tutorial not 100% clear

### ISSUE THREAD

**Issue by murphyk** [pytorch/tutorials#532]
I had a few minor suggestions about
https://pytorch.org/tutorials/beginner/blitz/autograd_tutorial.html

- In the "gradients" section, the following code is not explained

gradients = torch.tensor([0.1, 1.0, 0.0001], dtype=torch.float)
y.backward(gradients)

It is not obvious at first that y.backward(v) takes a directional derivative wrt v. Only after seeing this did I understand the mysterious comment "Because out contains a single scalar, out.backward() is equivalent to out.backward(torch.tensor(1))."

- The equation do/dx_i  =3/2 (x_i+2)  should be rewritten as do/dx_i = 1/4 * 6*(x_i + 2) for extra clarity.

**Comment by brianjo** [comment]
Thanks Kevin. Looking into making this clearer.

**Comment by brianjo** [comment]
Taking to Kevin, we found a change was not needed at this time. Closing.
